src/NetAnalyzers/Microsoft.CodeAnalysis.NetAnalyzers.md

```diff
@@ -1944,9 +1944,9 @@ Forward the 'CancellationToken' parameter to methods to ensure the operation can
 |CodeFix|True|
 ---
 
-## [CA2017](https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2017): Parameter count mismatch
+## [CA2017](https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2017): Parameter count mismatch or invalid braces in message template
```

Comment:
Hm, I thought only additional message will be enough, not sure we could/should change a title of an analyzer that released in .NET 6.0, maybe we should use a new rule ID? What you think @stephentoub @sharwell @mavasani?

Reply:
yeah i think that makes sense to handle it under a separate ID, it seems like from the localization generation "last in" wins, which had *completely* replaced the old title with the new. So i went the route of combining the title messages, keeping the "message" (or was it description?) distinct between the two descriptors

Reply:
Introduced as new `CA2023` instead of overloading `CA2017`.  `CA2023` is specific to orphaned unescaped braces

Reply:
Yes, new rule needs design review, though this is not like a completely new rule, so not sure, in case it needs API review we also need to move the issue back to runtime.

Reply:
looks like i can add a few more tests to help get coverage back up, but lmk if there's anything else needed from me